__tests__/app.test.js

```diff
@@ -187,4 +187,68 @@ describe("app", () => {
         });
     });
   });
+
+  describe("POST /api/reviews/:review_id/comments", () => {
+    // request body accepts  an object with username and body
+    // responds with POSTed comment
+    test("201: responds with newly created comment", () => {
+      // const requestBody = {
+      //   username: "TheCakeIsALie",
+      //   body: "A fun afternoon! Definitely recommend!!!",
+      // };
+      return request(app)
+        .post("/api/reviews/2/comments")
+        .expect(201)
```

Comment:
Lets do another 201 test that ignores unnecessary properties - for instance if we were to try and post  username: "philippaclaire9",
          body: "A fun afternoon! Definitely recommend!!!"
fruit: "apple"

The post would be successful still and would give 201 but the fruit information would be ignored